plugin/ToolBar.qml

```diff
@@ -35,6 +35,7 @@ PanelBackground {
             autoHideTimer.start()
         } else {
             autoHideTimer.stop()
+            _active = true
```

Comment:
Seems surprising from toolbar point of view, setting autoShowHide as false activates a hidden toolbar. If intention is to allow reactivating while toolbar is transitioning away, maybe check offset > 0?